topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -233,7 +239,30 @@ We have codified all of the dependencies you will need into a yaml file that `an
 >
 >    This will install all of the required modules for this training into the `roles/` folder. We choose to install to a folder to give you easy access to look through the different roles when you have questions on their behaviour.
 >
-> 4. Create the `hosts` inventory file if you have not done so, include a group for `[galaxyservers]` with the address of the host where you want to install Galaxy. Remember, if you are running ansible on the same machine as Galaxy will be installed to, you should set `ansible_connection=local`.
+> 4. Inspect the contents of the newly created `roles` directory in your working directory.
+{: .hands_on}
+
+> ### {% icon hands_on %} Hands-on: Configuration files
+>
+> 1. Create a `ansible.cfg` file (next to your playbook) to [configure settings](https://docs.ansible.com/ansible/latest/reference_appendices/config.html) like the inventory file (and save ourselves some typing!), or the Python interpreter to use:
+>
+>    > ```ini
+>    > [defaults]
+>    > interpreter_python = /usr/bin/python3
+>    > inventory = hosts
+>    > retry_files_enabled = false
```

Comment:
This is [the default](https://docs.ansible.com/ansible/latest/reference_appendices/config.html#retry-files-enabled) for a few versions now I think. Can't hurt though, of course.